manifests/gke/local-ssd-provision/local-ssd-provision.yaml

```diff
@@ -99,7 +99,7 @@ spec:
                 done
 
                 devs=$(echo "$ssd_mounts" | awk '{print $2}')
-                raid_dev=/dev/md0
+                raid_dev=/dev/mdx
```

Comment:
Is `/dev/mdx` the general case? maybe we should consider not using fixed path and list the devices first

Reply:
But we need to know what device we have chosen if this script is ran a 2nd time (avoiding data loss is the most important thing). We could write this to /var/raid_mount_point to